apps/portal/app/lib/services/points.ts

```diff
@@ -0,0 +1,137 @@
+import { pointsClient } from '@lib/graphql/client'
+
+export const GetRelicPointsDocument = {
+  query: `
+    query GetRelicPoints($address: String!) {
+      relic_points(where: {address: {_eq: $address}}) {
+        address
+        genesis_minter_points
+        snapshot_1_holder_points
+        snapshot_2_holder_points
+        total_relic_points
+      }
+    }
+  `,
+} as const
+
+export interface GetRelicPointsQuery {
+  relic_points: Array<{
+    address: string
+    genesis_minter_points: number
+    snapshot_1_holder_points: number
+    snapshot_2_holder_points: number
+    total_relic_points: number
+  }>
+}
+
+export interface GetRelicPointsQueryVariables {
+  address: string
+}
+
+export interface RelicPoints {
+  totalPoints: number
+  genesisPoints: number
+  snapshot1Points: number
+  snapshot2Points: number
+}
+
+export async function fetchRelicPoints(address: string): Promise<RelicPoints> {
+  console.log('fetchRelicPoints called with address:', address)
+
+  const data = await pointsClient.request<
+    GetRelicPointsQuery,
+    GetRelicPointsQueryVariables
+  >(GetRelicPointsDocument.query, {
+    address,
+  })
+
+  const points = data?.relic_points[0] ?? {
```

Comment:
Curious why this comes back as the 0 index?

Reply:
It returns an array, we provide the address, so we want the first one. Otherwise it returns all users relic points.